tests/test_basic_twoword_classifier.py

```diff
@@ -14,15 +14,16 @@ class BasicTwoWordClassifierTest(unittest.TestCase):
     def setUp(self):
         self.word1 = torch.from_numpy(np.array([[1, 0, 0]], dtype='float32'))
         self.word2 = torch.from_numpy(np.array([[0, 1, 0]], dtype='float32'))
-        self.inputdim = 6
-        self.hiddendim = 6
+        self.input_dim = 6
+        self.hidden_dim = 6
```

Comment:
if you change hidden_dim to a smaller value the test below will fail, because the hidden layer has been deleted and the shapes do not match anymore.

Reply:
should be okay now because I changed the mistake above